production-deployment-using-tiup.md

```diff
@@ -7,6 +7,10 @@ aliases: ['/docs/dev/how-to/deploy/orchestrated/tiup/','/docs/dev/tiflash/deploy
 
 # Deploy a TiDB Cluster Using TiUP
 
+> **Warning:**
+>
+> This is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment.
+
```

Comment:
This is added in the wrong location. @jackysp

Reply:
ah, yes...

Reply:
PTAL @yikeke